README.md

```diff
@@ -26,6 +26,9 @@ let props = {
   optionalNumber: ElementPropTypes.number,
```

Comment:
Check the current Readme and see the sections about the embeddable proptype. I think that for our new props additions is it worth to mention how they will work. For product for example, explain that it will trigger a product selector in the SD, and it will save in the config the product id.

Reply:
👍